code/game/machinery/computer/communications.dm

```diff
@@ -1,6 +1,8 @@
 //This file was auto-corrected by findeclaration.exe on 25.5.2012 20:42:31
 
 var/const/CALL_SHUTTLE_REASON_LENGTH = 12
+var/const/COMMUNICATION_COOLDOWN = 600
+var/const/COMMUNICATION_COOLDOWN_AI = 600
 

```

Comment:
Please add all three of theses as vars on the coms console.

Every global var or define slows down compile time by just a little bit more.